zarr/determine-chunk-shape.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        # we can set lon to 2 and lat to 1 since we expect twice as many chunks along the lon dimension as the lat dimension.
</pre><p>Maybe also mention what setting <code>-1</code> does? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>